hedera-node/hedera-app/src/test/java/com/hedera/node/app/workflows/TransactionScenarioBuilder.java

```diff
@@ -138,6 +139,8 @@ public static TransactionBody goodDefaultBody() {
                         .build())
                 .transactionFee(1L)
                 .transactionValidDuration(Duration.newBuilder().seconds(60).build())
+                .nodeAccountID(ALICE.accountID())
+                .utilPrng(UtilPrngTransactionBody.newBuilder().range(10).build())
```

Comment:
Why did you add this line? The `TransactionBody` is invalid if both `cryptoTransfer` and `utilPrng` are set.

Reply:
It was related to SingleTransactionRecordBuilder somehow but now it's not needed since there were some changes there and I refactored the tests. Removed 👍